packages/app/src/components/DBRowOverviewPanel.tsx

```diff
@@ -25,6 +25,10 @@ export function RowOverviewPanel({
   const { onPropertyAddClick, generateSearchUrl } =
     useContext(RowSidePanelContext);
 
+  const eventAttributesExpr =
+    source.eventAttributesExpression ??
+    (source.kind === SourceKind.Log ? 'LogAttributes' : 'SpanAttributes');
```

Comment:
Not sure if we want to use the hard-coded values. can we bypass it if the attribute expression isn't specified?